paddle/fluid/operators/run_program_op.h

```diff
@@ -232,10 +232,17 @@ class RunProgramOpKernel : public framework::OpKernel<T> {
 
     auto exe_ctx = exe.Prepare(*program, 0, skip_vars);
 
-    // get scope and clear old vars
-    framework::Scope &scope = *(out_scope_vec->front());
-    auto local_vars = scope.LocalVarNames();
-    scope.EraseVars(local_vars);
+    // NOTE(Aurelius84): While training some models, forward can be called many
+    // times and then apply
```

Comment:
Why is the comment line break so strange? Can you fix it?

Reply:
It seems modified by `pre-commit` tools.